packages/wallets/src/features/cfd/screens/MT5TradeScreen/MT5TradeScreen.tsx

```diff
@@ -1,36 +1,96 @@
+import { MarketTypeDetails, PlatformDetails } from '../../constants';
+import { useActiveWalletAccount, useMT5AccountsList } from '@deriv/api';
 
 const MT5TradeScreen = () => {
     const { isDesktop } = useDevice();
+    const { getModalState } = useModal();
+    const { data } = useMT5AccountsList();
+    const { data: activeWalletData } = useActiveWalletAccount();
+
+    const marketType = getModalState('marketType');
+    const platform = getModalState('platform');
+    const details = marketType
+        ? data?.filter(account => account.market_type === marketType)[0]
+        : data?.filter(account => account.platform === platform)[0];
 
     return (
         <div className='wallets-mt5-trade-screen'>
             <div className='wallets-mt5-trade-screen__details'>
                 <div className='wallets-mt5-trade-screen__details-description'>
                     <div className='wallets-mt5-trade-screen__details-description--left'>
-                        <DerivedMT5Icon />
-                        <div className='wallets-mt5-trade-screen__details-description-title'>
-                            <WalletText size='sm'>Derived</WalletText>
+                        {platform === 'mt5'
+                            ? MarketTypeDetails[marketType || 'all'].icon
+                            : PlatformDetails[platform || 'dxtrade'].icon}
+                        <div className='wallets-mt5-trade-screen__label'>
+                            <div className='wallets-mt5-trade-screen__title'>
+                                <WalletText lineHeight='3xs' size='sm'>
+                                    {platform === 'mt5'
+                                        ? MarketTypeDetails[marketType || 'all'].title
+                                        : PlatformDetails[platform || 'dxtrade'].title}{' '}
+                                    {details?.landing_company_short?.toUpperCase()}
+                                </WalletText>
+                                {activeWalletData?.is_virtual && <WalletListCardBadge isDemo label='virtual' />}
+                            </div>
                             <WalletText color='less-prominent' size='xs'>
-                                12345678
+                                {details?.loginid}
                             </WalletText>
                         </div>
                     </div>
-                    <WalletText weight='bold'>0.00 USD</WalletText>
+                    <div className='wallets-mt5-trade-screen__details-description--right'>
+                        <WalletText weight='bold'>{details?.display_balance}</WalletText>
+                        {!activeWalletData?.is_virtual &&
+                            details?.landing_company_short === 'svg' &&
+                            ['synthetic', 'financial'].includes(marketType || '') && (
+                                <div className='wallets-mt5-trade-screen__badge'>
```

Comment:
Can you use inlinemessage here instead? Maybe next PR

Reply:
yea sure, I'll include it in my next PR